securedrop/requirements/python3/securedrop-app-code-requirements.in

```diff
@@ -11,6 +11,7 @@ Flask-SQLAlchemy
 Flask-WTF>=1.0.0
 Flask>=2.0.2
 Jinja2>=3.0.0
+mako>=1.2.2
```

Comment:
I would prefer we didn't add indirect dependencies to the requirements.in file, as it makes it confusing in the future whether a dependency is actually needed by us when upstreams change their dependencies.

Now that requirements.txt has been bumped, you should be able to remove it from requirements.in, re-run the update thing, and it'll fix the `via` to be just alembic, but keep it at 1.2.2.

Reply:
@zenmonkeykstop: I removed the mako line from this file, re-ran `make update-python3-dependencies` twice, and both times it did not downgrade `mako`, so I think it's good now.